test/unit/shipment_packer_test.rb

```diff
@@ -36,6 +36,19 @@ def test_divide_order_with_single_line_into_two_packages
     end
   end
 
+  def test_divide_order_with_single_line_into_two_packages_max_weight_as_float
+    max_weight = 68038.8555
+
+    items = [{:grams => 45359, :quantity => 2, :price => 1.0}]
```

Comment:
just thought it looked prettier consistent 😎  but you're right, I'll change it :)